ruby/BUILD.bazel

```diff
@@ -3,6 +3,7 @@
 # See also code generation logic under /src/google/protobuf/compiler/ruby.
 
 load("@rules_pkg//:mappings.bzl", "pkg_files", "strip_prefix")
+load("@upb//cmake:build_defs.bzl", "generated_file_staleness_test")
```

Comment:
Seems a little weird for this to be in the cmake package, now might be a good time to move it given Adam's coming work to expand its usage?

Reply:
Makes sense, I shared a doc with you about this.